### ISSUE TITLE
Incorrect --destination description in help text

### ISSUE THREAD

**Issue by kevinoid** [jsdoc3/jsdoc#1156]
The help text from `jsdoc --help` includes

```
    -d, --destination <value>    The path to the output folder. Use
                                 "console" to dump data to the console.
                                 Default: ./out/
```

However, running `jsdoc -d console index.js` creates a directory named `console` which contains the output.  It appears that support for console output is template-dependent and `haruki` supports it while `default` does not.

Since not all templates (included or third-party) support this option and it is documented in `templates/haruki/README.md` for `haruki`, I'd suggest removing it from the help text.  If not, noting that support is template-specific would be helpful for new users.

Thanks,
Kevin

**Comment by kevinoid** [comment]
I just noticed the [description on the website](http://usejsdoc.org/about-commandline.html).  Using that in the helptext would probably make the most sense.

**Comment by hegemonic** [comment]
I'm pretty sure that hardly anyone uses the `haruki` template, so I just removed the erroneous text from the description of the `-d` flag. The fix will be included in JSDoc 3.5.3.
